### ISSUE TITLE
a-a-install-debuginfo fails to find some corresponding debuginfo packages

### ISSUE THREAD

**Issue by mcatanzaro** [abrt/abrt#1533]
```
--- Running analyze_CCpp ---
Ok to upload core dump? (It may contain sensitive data). If your answer is 'No', a stack trace will be generated locally. (It may download a huge amount of data). 'YES'
Querying server settings
Preparing an archive to upload
You are going to upload 21.1 MiB. Continue? 'YES'
Uploading 21.1 MiB
Upload successful
Retrace job started
Analyzing crash data
.........................................
Preparing environment for backtrace generation
.......
Retrace job failed
Retrace failed. Try again later and if the problem persists report this issue please.
[2020-10-08 15:31:48] [I] Analyzing crash data

Do you want to generate a stack trace locally? (It may download a huge amount of data but reporting can't continue without stack trace). 'YES'
Analyzing coredump 'coredump'
Cleaning cache...
Cache cleaning has finished
Coredump references 204 debuginfo files, 224 of them are not installed
```

Something smells off here, because 224 > 204. This is a gnome-shell crash, so presumably you can send it SIGSEGV to reproduce. Anyway, continuing:

```
Coredump references 204 debuginfo files, 224 of them are not installed
Initializing package manager
Setting up repositories
Looking for needed packages in repositories
Can't find packages for 204 debuginfo files
Packages to download: 11
```

ABRT should not fail to find required debuginfo packages. Instead of ignoring this problem, it should automatically report a bug to investigate why 204 debuginfo files are missing.

**Comment by mgrabovsky** [comment]
The first problem has been fixed in 12642e64ed8d1493d919d80c4bbea634beed3845, which has not yet been released.

I'll look into the second point.

**Comment by msrb** [comment]
ABRT now relies on debuginfod and it doesn't try to install individual debuginfo packages.
